Himalayan_Odyssey/HO.md

```diff
@@ -0,0 +1,26 @@
+Dreams to Memories – Wisdom in Ladakh.To read this click [here](language.md)
+A ride to ladakh requires a lot of preparation and begins with convincing folks back home and also at the office. 
+As a popular quote says “Four Wheels move the body and two wheels move the soul” just riding 2500 Kms in 15 days was not the only objective
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.